### DISCUSSION TITLE
Geomap: Autozoom

### DISCUSSION THREAD

**Discussion by UlrichThiess** [grafana/grafana#37376] [Dashboards, Panels & Visualizations]
Don't forget Autozoom please.

**Comment by mertbakir** [comment]
Not sure but I think this is relevant: https://github.com/grafana/grafana/discussions/44850

**Comment by UlrichThiess** [comment]
It works. Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
